server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestGetScriptContextAction.java

```diff
@@ -22,7 +22,6 @@
 
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 
-@ServerlessScope(Scope.PUBLIC)
```

Comment:
If they are used by Kibana then they need to be Scope.INTERNAL?

Reply:
from what I spoke with kibana team, these are NOT used
only `/_scripts/painless/_execute` is used